help/keyboard-shortcuts.md

```diff
@@ -166,6 +166,8 @@ in the Zulip app to add more to your repertoire as needed.
   don't show up in any views (including **All messages**), and don't contribute
   to unread counts. Read more about [muting or unmuting topics](/help/mute-a-topic).
 
+* **View edit history**: <kbd>H</kbd> - Depends on the organization permissions.
```

Comment:
Maybe squash this into your docs commit? It seems like a related change to me.

Reply:
Done, thanks for the suggestion :)